models/validators/MethodValidator.cfc

```diff
@@ -38,11 +38,6 @@ component extends="BaseValidator" accessors="true" singleton {
 			return true;
 		}
 
-		// return true if no data to check, type needs a data element to be checked.
-		if ( isNull( arguments.targetValue ) || isNullOrEmpty( arguments.targetValue ) ) {
-			return true;
-		}
-
 		// Validate via method
```

Comment:
The only issue I see here, is that if `arguments.targetValue` is null, then line 46 will fail

Reply:
I'm not sure I follow. If `arguments.targetValue` is null, the method will return `true`, so it should never get to line 46.